lightning/src/ln/onchaintx.rs

```diff
@@ -152,31 +152,31 @@ macro_rules! subtract_high_prio_fee {
 	($logger: ident, $fee_estimator: expr, $value: expr, $predicted_weight: expr, $used_feerate: expr) => {
 		{
 			$used_feerate = $fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::HighPriority);
-			let mut fee = $used_feerate * ($predicted_weight as u64) / 1000;
-			if $value <= fee {
+			let mut fee = $used_feerate * ($predicted_weight as u32) / 1000;
```

Comment:
Probably fine, but probably use a u64 (and would probably reduce diff size further down in the file).